src/components/DateTimePicker/_helpers/custom_locale/hy/_lib/formatDistance/index.js

```diff
@@ -0,0 +1,78 @@
+var formatDistanceLocale = {
+  lessThanXSeconds: {
+    one: 'ավելի քիչ քան 1 վայրկյան',
```

Comment:
Initially i went with "1 վայրկանից պակաս" myself, then i realized this will be used in conjunction with "առաջ / հետո", and  "ավելի քիչ քան 1 վայրկյան առաջ" sounded less awkward than  "1 վայրկյանից պակաս առաջ"